src/applications/ivc-champva/10-10D/containers/IntroductionPage.jsx

```diff
@@ -148,7 +148,7 @@ const IntroductionPage = props => {
 };
 
 IntroductionPage.propTypes = {
-  isLoggedIn: PropTypes.func,
+  isLoggedIn: PropTypes.bool,
```

Comment:
Fixing proptypes warning since this should be a boolean. Did this everywhere this prop is used.